splunk_eventgen/lib/eventgenoutput.py

```diff
@@ -98,6 +98,7 @@ def flush(self, endOfInterval=False):
             outputer = self.outputPlugin(self._sample, self.output_counter)
             outputer.updateConfig(self.config)
             outputer.set_events(q)
+            del q
```

Comment:
why do we need to to delete `q` and `outputer` explicitly?

Reply:
Previously I tried this to check if memory usage will downgrade if explicitly delete. It is better but we still got OOM error. I still keep this change.